assets/src/styles/blocks/Covers/ContentCovers.scss

```diff
@@ -81,42 +81,20 @@
         text-decoration: underline;
       }
 
-      a {
-        color: $blue;
-      }
-
-      h4 {
-        font-size: $font-size-xxs;
-        line-height: 1.2;
-        font-weight: 500;
+      h5 {
```

Comment:
Nice that we can achieve consistency by just removing this rules :+1: This does have potential to break child themes, so we probably should set up the test instance with CH, NL and Nordics child themes so we can check if they need to do changes for this.

Reply:
Good point @Inwerpsel ! I'll check that out

Reply:
Checked with NL and Nordics themes, seems fine 👌 However CH theme might need small tweaks so I've messaged Stefan about it!